docs/source/alfacase/case_description.example.rst

```diff
@@ -3,5 +3,439 @@
+
+.. note::
+
+    Check the :ref:`alfacase-reference-pipe-section` section for a detailed description of the attributes available.
+
+.. rubric:: Example
+
+
+.. code-block:: python
+
+    from alfasim_sdk import (
+        CaseDescription,
+        PipeDescription,
+        ProfileDescription,
+        LengthAndElevationDescription,
+        PipeSegmentsDescription,
+    )
+
+    CaseDescription(
+        # Omitting fields that configure Options, Pvt and Node
+        pipes=[
+            PipeDescription(
+                name="pipe",
+                source="Inlet",
+                target="Outlet",
+                profile=ProfileDescription(
+                    length_and_elevation=LengthAndElevationDescription(
+                        length=Array([0.0, 15.0, 30.0, 30.0, 15.0], "m"),
+                        elevation=Array([0.0, 15.0, 30.0, 30.0, 15.0], "m"),
+                    ),
+                ),
+                segments=PipeSegmentsDescription(
+                    start_positions=Array([0.0], "m"),
+                    diameters=Array([0.1], "m"),
+                    roughnesses=Array([5e-05], "m"),
+                ),
+            ),
+        ],
+    )
+
+
+
+
+Output
+------
+
+The final step for our example will add a trend and a profile for our project.
+
+As indicated on :class:`CaseDescription <alfasim_sdk.CaseDescription>`, the ``outputs`` field must be filled with
+the definition of :class:`CaseOutputDescription <alfasim_sdk.CaseOutputDescription>` which allows the configuration of ``trends``
+and ``profiles``.
+
+.. note::
+
+    Check the :ref:`alfacase-reference-output-section` section for a detailed description of each output type, that
+    shows all the available curves that can be used.
+
+.. rubric:: Example
+
+.. code-block:: python
+
+    from alfasim_sdk import (
+        CaseDescription,
+        CaseOutputDescription,
+        TrendOutputDescription,
+        OutputAttachmentLocation,
+        ProfileOutputDescription,
+    )
+
+    CaseDescription(
+        # Omitting fields that configure Options, Pvt, Node and Pipes
+        outputs=CaseOutputDescription(
+            trends=[
+                TrendOutputDescription(
+                    element_name="pipe",
+                    curve_names=["oil mass flow rate"],
+                    position=Scalar(100.0, "m"),
+                    location=OutputAttachmentLocation.Main,
+                )
+            ],
+            trend_frequency=Scalar(0.1, "s"),
+            profiles=[
+                ProfileOutputDescription(
+                    curve_names=["pressure"],
+                    element_name="pipe",
+                    location=OutputAttachmentLocation.Main,
+                )
+            ],
+            profile_frequency=Scalar(0.1, "s"),
+        ),
+    )
+
+
+Full Case
+---------
+
+This section brings together all the previous sections, showing the full example that can be now used
```

Comment:
This is a bit confusing, because just the classes are not enough to be imported by the application; I suggest to just merge this section and the last.

Reply:
Sure I will change it to:


```
This section brings together all the previous sections, showing the full example of a Description with a project configuration being converted to a ``.alfacase`` file, ready to be imported by the application.

Reply:
Yes, good idea.